src/gui/uithememanager.cpp

```diff
@@ -102,30 +117,23 @@ void UIThemeManager::applyStyleSheet() const
     qApp->setStyleSheet(qssFile.readAll());
 }
 
-QIcon UIThemeManager::getIcon(const QString &iconId) const
-{
-    return getIcon(iconId, iconId);
-}
-
 QIcon UIThemeManager::getIcon(const QString &iconId, const QString &fallback) const
 {
 #if (defined(Q_OS_UNIX) && !defined(Q_OS_MACOS))
     if (m_useSystemTheme) {
         QIcon icon = QIcon::fromTheme(iconId);
         if (icon.name() != iconId)
-            icon = QIcon::fromTheme(fallback, QIcon(IconProvider::instance()->getIconPath(iconId)));
+            icon = QIcon::fromTheme(fallback, QIcon(getIconPathFromResources(iconId, fallback)));
         return icon;
     }
-#else
-    Q_UNUSED(fallback)
```

Comment:
why use fallback now?

kinda helps in the direct adoption of Linux icon themes.